.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
+  extends: [
+    "airbnb-typescript",
+    "prettier",
+    "prettier/@typescript-eslint",
+    "prettier/react",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:react/recommended"
+  ],
+  parserOptions: {
+    project: "./tsconfig.json",
+    tsconfigRootDir: __dirname
+  },
+  rules: {
+    "@typescript-eslint/no-use-before-define": [
+      "error",
```

Comment:
This plugin has 4 out of 5 checks, but we are silencing 4. What is the use of this plugin then? Unless you set some to true

Reply:
This is actually really true, we're gonna fix this up so thats it only for variables so only 1/5 is silenced